### DISCUSSION TITLE
How to add class to the Fancybox container?

### DISCUSSION THREAD

**Discussion by oshumelka** [fancyapps/ui#451] [Q&A] [answered]
Are there any built-in options to add a class to the Fancybox container ('.fancybox __ container')? baseClass equivalent in Fancybox 3.

**Comment by fancyapps** [comment, marked as answer]
Hi,

There is `mainClass` option for v4 and v5. Here is a basic example of how to create a light theme - https://jsfiddle.net/r7zm3p62/

**Reply by oshumelka** [reply]
Thanks a lot!!!
